samples/android/Continote/app/src/main/java/com/firebasecontinue/sample/continote/EditNoteActivity.java

```diff
@@ -100,44 +173,180 @@ protected void handleUserSignedOut() {
     }
 
     /**
+     * Handles when the Note the user wants to edit has been found.
```

Comment:
Add more detail of what this method does (since it does quite a few things). e.g. "Configure the UI to be ready to edit a note" or similar.